code/modules/projectiles/guns/ballistic/toy.dm

```diff
@@ -31,6 +32,9 @@
 	magazine = new /obj/item/ammo_box/magazine/toy/pistol/riot(src)
 	return ..()
 
+/obj/item/gun/ballistic/automatic/pistol/toy/riot/clandestine
+	projectile_damage_multiplier = 1.6
```

Comment:
40 is a very important threshold for stamloss, as 40/75 are big stamina breakpoints. 35 will still stamcrit in 3 shots but won't apply slowdown after the first/increase slowdown after the second shot.
```suggestion
/obj/item/gun/ballistic/automatic/pistol/toy/riot/clandestine
	projectile_damage_multiplier = 1.4
```

Reply:
Fair enough.